### PR TITLE
Daily treasury graph

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
# Daily treasury graph

_On the Treasury page, we need to design a graph that gives the data of the daily treasury value.

this graph should be positioned above the NFT images and the size of the nft should be relatively reduced.

fixes https://github.com/EveripediaNetwork/issues/issues/1542
fixes https://github.com/EveripediaNetwork/issues/issues/1555

### CODE REVIEW

**Comment by kesar** [comment]
you should consider having linters properly installed on your IDE / pre commit hooks :P

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
Doesn't look good on mobile 

- shouldn't this be in dollars?

-then the token symbols are missing on the vertical sides of the graph (noticed this on the dashboard too for iq staked)

- the nfts should have similar animation to wikis on iq wiki where they change at interval.

File: src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -139,150 +162,154 @@ const Home: NextPage = () => {
...
+            <GraphPeriodWrapper getRootProps={getRootProps}>
+              {GRAPH_PERIODS.map((btn) => {
+                return (
+                  <GraphPeriodButton
+                    key={btn.period}
+                    label={btn.label}
+                    {...getRadioProps({ value: btn.period })}
+                  />
+                )
+              })}
```
Comment (OleanjiKingCode) [inline src/app/dashboard/page.tsx:286]: The graph component from line 167- 288 can be reduced using the  <GraphComponent/>

File: src/utils/treasury-utils.ts [hunk 2]
```diff
@@ -139,11 +139,11 @@ export const getTreasuryDetails = async () => {
...
 
-export const getCurrentPageData = (
-  currentPage: number,
-  dataArray: TreasuryTokenType[],
-) => {
-  const startIndex = (currentPage - 1) * 6
-  const endIndex = startIndex + 6
-  return dataArray.slice(startIndex, endIndex)
-}
+// export const getCurrentPageData = (
```
Comment (OleanjiKingCode) [inline src/utils/treasury-utils.ts:142]: this part of the code is commented not sure we are still using this

File: src/theme/semantic-tokens.ts [hunk 3]
```diff
@@ -10,6 +10,10 @@ export const semanticTokens: SemanticTokens = {
       default: 'white',
       _dark: 'gray.800',
     },
+    toolTipBg: {
+      default: 'whiteAlpha.900',
```
Comment (OleanjiKingCode) [inline src/theme/semantic-tokens.ts:14]: The color for this toolTipBg already exists in line 147-150 and the dark is gray 700 from the design

**Comment by OleanjiKingCode** [comment]
- The graph and nft carousel look in a lesser screen size


image


- Different price timestamps options than in design

image


- Timestamp price options doesn't change the graph look when clicked

image

image

**Review by OleanjiKingCode — APPROVED** [review]

**Pull request merged**